cmake/luajit.cmake

```diff
@@ -67,6 +67,6 @@ if (NOT lua_POPULATED)
 endif()
 
 add_library(luajit INTERFACE)
-target_include_directories(luajit INTERFACE ${luajit_BINARY_DIR}/include)
+target_include_directories(luajit INTERFACE ${luajit_BINARY_DIR}/include ${luajit_SOURCE_DIR}/src)
```

Comment:
Why to include LuaJIT src here?

Reply:
It appears to be luajit.h This automatically generated file is not found in the new version of luajit. I'll check it out when I have time.